Update_item_demo.py

```diff
@@ -0,0 +1,17 @@
+import requests
```

Comment:
These files are really useful for testing, but it's a good habit to clear them out once you've got the app working since that will be the source of truth for how to handle this moving forwards